### ISSUE TITLE
CommandBar failing accessibility fast pass for rule "aria-required-children"

### ISSUE THREAD

**Issue by HarshUpadhyay-MSFT** [microsoft/fluentui#15052]
### Environment Information

- **Package version(s)**: office-ui-fabric-react version 7.117.0
- **Browser and OS versions**: edge chromium Version 85.0.564.51 (Official build) (64-bit), Windows 10 OS

### Describe the issue:
Running accessibility fast pass in any page causes failures on "aria-required-children" rule for the CommandBar component. This should not happen as default role is `menuitem` for commandBar items

### Please provide a reproduction of the issue in a codepen:

https://codepen.io/dzearing/pen/PoPEyBR

#### Actual behavior:

N/A
#### Expected behavior:

N/A
### Documentation describing expected behavior

N/A

**Comment by paulgildea** [comment]
@HarshUpadhyay-MSFT Thanks for the feedback.

Dupe of #14805
